target/linux/ramips/base-files/lib/ramips.sh

```diff
@@ -277,6 +277,9 @@ ramips_board_detect() {
 	*"Mercury MAC1200R v2")
 		name="mac1200rv2"
 		;;
+	*"Mi Router 3 Pro")
```

Comment:
with the dt-based boardname discovery, It should be possible to drop the whole hunk.

Reply:
This one I'm not sure I understand... do you want me to not change this file at all?

Reply:
> This one I'm not sure I understand... do you want me to not change this file at all?

yes exactly :+1. Sorry for my late night denglish. 

You can safely drop the target/linux/ramips/base-files/lib/ramips.sh change. Everything should still work, because thedevice's /tmp/sysinfo files will be provided by the common DeviceTree-based extraction script in

https://github.com/openwrt/openwrt/blob/master/package/base-files/files/lib/preinit/02_sysinfo

which is available on all devices since it's part of the "base-files" package.

Reply:
OK. Fixed.